tests/operator_test.go

```diff
@@ -2501,6 +2501,9 @@ spec:
 			}, 60*time.Second, 1*time.Second).Should(BeTrue())
 
 			patchKvInfra(nil, false, "")
+
+			By("Verifying that all pods are ready to avoid flackiness in the next test")
+			allPodsAreReady(originalKv)
```

Comment:
We already do it [here](https://github.com/kubevirt/kubevirt/blob/816fae169cc53e822ac89ad92ae4a4d2296684bc/tests/operator_test.go#L1047)